### PR TITLE
Fix/1043/stucks reprod

### LINKED ISSUE #1079: MWoffliner 1.10.6 dying because of Mediawiki API timeouts
See for example https://farm.openzim.org/recipes/wikipedia_en_all_maxi

### PR DESCRIPTION
**Pull request by midik**
fixes #1079 
add debug logging that probably help to solve #1043

### CODE REVIEW

File: src/Downloader.ts [hunk 1]
```diff
@@ -466,9 +467,13 @@ class Downloader {
           resolve(val);
         }
       });
-      call.retryIf((err: any) => err.response && err.response.status !== 404);
+      // todo pull this logic up to the class
```
Comment (bakshiutkarsha) [inline src/Downloader.ts:470]: As you have already mentioned it as Todo, I think it's better if we can create a ticket for this to note it.

**Comment by bakshiutkarsha** [comment]
@midik the codecov report for the PR seems to decrease the coverage.

File: src/Downloader.ts (hunk 1 above)
Comment (midik) [reply src/Downloader.ts]: You're right, however, that's just a kind of refactoring so I don't think it deserve the ticket. I redesigned that.

**Comment by midik** [comment]
@bakshiutkarsha slightly stepped up the coverage (technically) by covering redisKvs.keys(). We'll rethink the testing strategy soon, once we get couple critical bugs fixed.

File: src/Downloader.ts (hunk 1 above)
Comment (bakshiutkarsha) [reply src/Downloader.ts]: Great 👍

**Pull request merged**